<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tomatoClock</class>
 <widget class="QDialog" name="tomatoClock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="current_clock">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>current task:</string>
   </property>
  </widget>
  <widget class="QLabel" name="tomato_number">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>35</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>tomato number:</string>
   </property>
  </widget>
  <widget class="QLabel" name="task_value">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>250</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="tomatoNum">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>35</y>
     <width>250</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="remained_time">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>210</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>29:59</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="state">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>270</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>work</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="tomato_graphic">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>tomato_graphic</zorder>
  <zorder>current_clock</zorder>
  <zorder>tomato_number</zorder>
  <zorder>task_value</zorder>
  <zorder>tomatoNum</zorder>
  <zorder>remained_time</zorder>
  <zorder>state</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
